packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -440,6 +441,21 @@ const DataManagerProvider = ({
           initialData.contentType
         );
 
+        const isValidSchema = validateSchema(contentType);
+
+        if (!isValidSchema) {
+          toggleNotification({
+            type: 'warning',
+            message: {
+              id: getTrad('notification.error.dynamiczone-min.validation'),
+              defaultMessage:
+                'At least one component is required in a dynamic zone to be able to save a content type',
```

Comment:
Since we only have a verification on the DZ length it's OK. I guess that in the long term we'll replace this hardcoded message by something returned by the `isValidSchema` or smth like that?

Reply:
Yes indeed I wanted to introduce a schema validation with yup but I thought it was too overkill for this, plus according to the US we alway display the same message.